js/interactive-guides/microprofile-config/microprofile-config-callback.js

```diff
@@ -160,7 +160,7 @@ var microprofileConfigCallBack = (function() {
             //   <space or newline here>
             // @Inject @ConfigProperty(name=\"download-url\", defaultValue=\"ftp://music.com/us/download\")
             // private String downloadUrl;
-            var contentToMatch = "[\\s\\S]*private Config config;\\s*@Inject\\s*@ConfigProperty\\s*\\(([\\s\\S]*)\\)\\s*private String downloadUrl;";
+            var contentToMatch = "[\\s\\S]*private Config config;\\s*@Inject\\s*@ConfigProperty\\s*\\(([\\s\\S]*)\\)\\s*private String download-url;";
```

Comment:
Why are you changing it to download-url? It is a java variable and it is usually downloadUrl.